data/pages.yaml

```diff
@@ -1,51 +1,67 @@
-  #Ansible installation
-  install_action: Go to the installation guide
-  #Ansible user
+    # Labels provide meaningful descriptions of elements for accessibility purposes.
+    label: Button group that links to documentation for new Ansible users.
+    learn: Understand the fundamentals of Ansible automation
+    setup: Set up an Ansible automation project
+    start: Write your first playbook in a few easy steps
+  # Ansible installation
+  install_action: Find more installation options
+  # Ansible user
   user_heading: Ansible users
-  user_base_body: Ansible playbooks are blueprints for automating software deployment and system configuration as well as orchestrating complex operations.
-  user_base_actions:
-    #Labels provide meaningful descriptions of elements for accessibility purposes.
-    action_label: Button group that links to base Ansible user documentation.
-    action_one: Start writing Ansible playbooks
-    action_two: Learn which plugins to use for your tasks
-    action_three: Build inventory files to manage multiple hosts
+  user_body: Ansible playbooks are blueprints for automating software deployment and system configuration as well as orchestrating complex operations.
+  user_actions:
+    # Labels provide meaningful descriptions of elements for accessibility purposes.
+    label: Button group that links to base Ansible user documentation.
+    automate: Start writing Ansible playbooks
+    evaluate_plugins: Learn which plugins to use for your tasks
+    build_inventory: Build inventory files to manage multiple hosts
   user_resources_heading: User resources
   user_resources_body: Structure your project with roles, improve content quality, re-use existing automation, and more.
-  user_resources_action: Find user resources
-  #Ansible developer
+  user_resources_action: Find more resources for Ansible users
+  # Ansible developer
   developer_heading: Ansible developers
-  developer_base_body: Ansible is open source. You can add code that fixes bugs, implements new features, or extends the benefits of simple, agentless automation.
-  developer_base_actions:
-    #Labels provide meaningful descriptions of elements for accessibility purposes.
-    action_label: Button group that links to base Ansible developer documentation.
-    action_one: Set up your development environment
-    action_two: Review the Ansible contributor guidelines
-    action_three: Write custom modules or plugins
+  developer_body: Ansible is open source. You can add code that fixes bugs, implements new features, or extends the benefits of simple, agentless automation.
+  developer_actions:
+    # Labels provide meaningful descriptions of elements for accessibility purposes.
+    label: Button group that links to base Ansible developer documentation.
+    setup: Set up your development environment
+    guidelines: Review the Ansible contributor guidelines
+    start: Write custom modules or plugins
```

Comment:
looks like the `learn` is missing here? There's no longer a link to the architecture showing up tho it is listed at https://github.com/ansible/jinja-docsite/pull/33/files#diff-e63e46cde797a05be9a5a69b973f28a8c8d68cb2b161a39f817dd8c637af0facR22 (links.yaml)